DEVELOPMENT.md

```diff
@@ -135,11 +135,11 @@ Kubernetes cluster in your designated environment, if necessary.
 ### Deploy Istio
 
 ```shell
-kubectl apply -f ./third_party/istio-1.3-latest/istio-crds.yaml
+kubectl apply -f ./third_party/istio-1.4-latest/istio-crds.yaml
```

Comment:
Why not use the istio-1.5-latest in master branch?

Reply:
I believe there were flakiness issues with 1.5
https://github.com/knative/serving/pull/7274